### ISSUE TITLE
Color codes do not work in windows

### ISSUE THREAD

**Issue by 00001** [ultralist/ultralist#143]
Reading todo contents is more or less impossible when using 'xterm -bg lemonchiffon', or any bright background.

Is there a way to change or disable colors, without affecting the terminal?

**Comment by robertpateii** [comment]
Here's how I disabled colors in case any window users are desperate:
```
diff --git a/ultralist/screen_printer.go b/ultralist/screen_printer.go
index 7a9c429..4fc459c 100644
--- a/ultralist/screen_printer.go
+++ b/ultralist/screen_printer.go
@@ -37,6 +37,7 @@ type ScreenPrinter struct {

 // NewScreenPrinter creates a new screeen printer.
 func NewScreenPrinter() *ScreenPrinter {
+       color.NoColor = true // disables colorized output
        w := new(tabwriter.Writer)
        w.Init(os.Stdout, 0, 8, 0, '\t', 0)
        formatter := &ScreenPrinter{Writer: w}
```
If you used **go get** to install this like I did, you'll find the source in ~\go\src\github.com\ultralist\ultralist. Modify that file, run a **go build**, then copy the new ultralist.exe to ~\go\bin\, and then it should start working without colors immediately.

**Comment by bothzoli** [comment]
i don't know the first thing about Go, but i managed to resolve this with the following changes:
```diff
diff --git a/ultralist/screen_printer.go b/ultralist/screen_printer.go
index 9e8f89c..acc03ff 100644
--- a/ultralist/screen_printer.go
+++ b/ultralist/screen_printer.go
@@ -7,6 +7,7 @@ import (
        "strconv"
        "strings"
        "time"
+       "text/tabwriter"

        "github.com/cheynewallace/tabby"
        "github.com/fatih/color"
@@ -48,7 +49,7 @@ func (f *ScreenPrinter) Print(groupedTodos *GroupedTodos, printNotes bool) {
        }
        sort.Strings(keys)

-       tabby := tabby.New()
+       tabby := tabby.NewCustom(tabwriter.NewWriter(color.Output, 0, 0, 2, ' ', 0))
        tabby.AddLine()
        for _, key := range keys {
                tabby.AddLine(cyan.Sprint(key))
```
maybe based on this you could solve this the idiomatic way...

**Comment by gammons** [comment]
@bothzoli I'll create a PR based upon this change and test it out.

**Comment by bothzoli** [comment]
thanks a lot.
i'm not sure how it'd affect other OS-s, and also i know almost nothing about Go that's why i didn't open a PR myself :)
but i hope you'll be able to resolve this issue with the above info.

**Comment by gammons** [comment]
@bothzoli give that PR a review.  if it looks good I'll merge it and build a release!

**Comment by gammons** [comment]
@00001 @cvrn @bothzoli @robertpateii just released 0.9.4 which includes this fix https://github.com/ultralist/ultralist/releases
